pkg/scanner/gitleaks.go

```diff
@@ -139,7 +139,7 @@ func (g *Gitleaks) walkScan(detector *detect.Detector, scanResource resource.Res
 				return nil
 			}
 			if mimetype.MIME.Type == "application" {
-				logger.Error("skipping binary file: path=%q", path)
+				logger.Info("skipping binary file: path=%q", path)
```

Comment:
Changed this to Info as it is not really an error, it is more a known limitation.
Thoughts?

Reply:
I think `Warning` would be good since it's working as expected but the user may not expect the file to be skipped.